doc/explanation/networking.md

```diff
@@ -0,0 +1,105 @@
+
+- **Dynamic address management**: Manages DHCP and DNS within the cluster.
+
+- **Access control lists**: Define security policies for network access.
+
+- **Resiliency**: OVN virtual routers are highly available; they can automatically fail over to another cluster member as needed.
+
+(exp-networking-ovn-architecture)=
+### Networking architecture
+
+OVN requires the use of cloud management software (CMS) to connect OVN logical networks to a physical network. Since MicroCloud is built on {ref}`LXD clusters <lxd:exp-clusters>`, LXD acts as the CMS for MicroCloud, enabling the connection of an OVN network to an existing managed {ref}`lxd:network-bridge` or {ref}`lxd:network-physical` for external connectivity.
+
+The following figure shows the OVN network traffic flow in a MicroCloud/LXD cluster:
+
+```{figure} /images/ovn_networking_1.svg
+:width: 100%
+
+OVN networking (single network)
+```
+
+In MicroCloud, the OVN network connects all cluster members. Intra-cluster traffic (communication between members of the same cluster, also known as OVN east/west traffic) travels through an OVN tunnel over a designated {abbr}`NIC (Network Interface Card)` (`eth1` in the figure above).
+
+For external connectivity, the OVN network connects to an uplink network via a virtual router on a designated NIC (`eth0` in the figure above). The virtual router is active on only one cluster member at a time but can migrate between cluster members as needed, ensuring all members maintain uplink access.
+
+An instance (container or virtual machine) within a cluster member connects to the OVN network via its virtual NIC. Through the OVN network, it can communicate with the uplink network. Traffic between instances and the uplink network is called OVN north/south traffic.
+
+The strengths of using OVN become apparent when considering a networking architecture with more than one OVN network:
+
+```{figure} /images/ovn_networking_2.svg
+:width: 100%
+
+OVN networking (two networks)
+```
+
+Both OVN networks depicted are completely independent. Both networks are available on all cluster members (with each virtual router active on one random cluster member). Any instance can use either of the networks, and the traffic on one network is completely isolated from the other network.
+
+See the {ref}`LXD documentation on OVN networks <lxd:network-ovn>` for more information.
+
+(exp-networking-ovn-underlay)=
+### Dedicated underlay network
+
+During {ref}`MicroCloud initialisation <howto-initialise>`, you can choose to {ref}`configure a dedicated underlay network for OVN traffic <howto-ovn-underlay>`. This requires an additional network interface on each cluster member.
+
+A dedicated underlay network serves as the physical infrastructure over which the virtual (overlay) network is constructed. While optional, it offers several benefits:
```

Comment:
Given my comment here https://github.com/canonical/microcloud/pull/633/commits/ecf96744016ca75a15c98162ef41c43a3a267fc3#r1954242012 suggest we add the word "potential":

> it offers several potential benefits:

As "Reduced congestion" and "Optimised performance" wont necessarily be true if using a shared physical link with VLANs on for "Traffic isolation".